README.md

```diff
@@ -159,6 +159,7 @@ The environment variables are analogous to the script's command line arguments.
 | API_KEY            | no | The Immich API Key to use. Either `API_KEY` or `API_KEY_FILE` must be specified. The `API_KEY` variable takes precedence for ease of manual execution, but it is recommended to use `API_KEY_FILE`. 
 | API_KEY_FILE       | no | An absolute path (from the root of the container) to a file containing the Immich API Key. The file might be mounted into the container using a volume (e.g. `-v /path/to/api_key.secret:/immich_api_key.secret:ro`). The file must contain only the value. |
 | CRON_EXPRESSION    | yes | A [crontab-style expression](https://crontab.guru/) (e.g. `0 * * * *`) to perform album creation on a schedule (e.g. every hour). |
+| RUN_IMMEDIATELY    | no | Set to `true` to run the script right away, after running once the script will automaitcally run again based on the CRON_EXPRESSION |
```

Comment:
Little typo: `automaitcally` should be `automatically`

Reply:
oops